### ISSUE TITLE
simplesnapwrap not working on MacOS

### ISSUE THREAD

**Issue by tomhoover** [jgoerzen/simplesnap#22]
1. When installing OpenZFS and the gnu utilities (gdate, ghead, ggrep & gsed) via [Homebrew](https://brew.sh), all executables are placed in the /usr/local/bin directory. As the default PATH used when executing simplesnapwrap doesn't include the /usr/local/bin directory, none of the executables are found by the script.
2. The script doesn't use `ghead` (if available). The default `head` command in MacOS is not gnu compatible; therefore, the command `head -n -1` returns an error.

Pull request is nearly complete, and should be uploaded tomorrow.

**Comment by clarkema** [comment]
I don't run ZFS on macos, so can't test directly, but this PR looks like it will only work on Intel macs. Homebrew uses `/opt/homebrew` on Apple silicon (see https://docs.brew.sh/FAQ#why-is-the-default-installation-prefix-opthomebrew-on-apple-silicon)

If you're running in a Homebrew-aware shell, you can check the environment variable `$HOMEBREW_PREFIX` to find the root of the Homebrew installation.

**Comment by tomhoover** [comment]
Good point. When I submitted this PR, there was no such thing as a Mac running Apple silicon. :-)

@jgoerzen, although I no longer run simplesnapwrap, I'd be happy to make the appropriate change and test sometime this weekend. 

-- 
  Tom Hoover
  ***@***.***

**Comment by tomhoover** [comment]
@clarkema , I just submitted https://github.com/jgoerzen/simplesnap/pull/31 which works with both Apple Silicon and Intel.
